File: 1-TwoSum/1-TwoSum.py
# Last updated: 4/17/2026, 12:40:28 AM
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        seen = {}
        for i,v in enumerate(nums):
            compliment = target - v 
            if compliment in seen:
                return [i,seen[compliment]]
            seen[v] = i






        

File: 1-TwoSum/1-TwoSum.ts
// Last updated: 4/17/2026, 12:40:30 AM
function twoSum(nums: number[], target: number): number[] {


    for(let i=0;i<nums.length;i++)
    {
        for(let j=i+1;j<nums.length;j++)
        {
            if(nums[i] + nums[j] == target)
            {
              
                return  [i,j];
            }
        }
    }    

};